tests/test_env_path_safety.py

```
@@ -0,0 +1,54 @@
+    def test_outside_home_rejected_by_default(self, monkeypatch, caplog):
+        from siege_utilities.config.paths import _env_path
+        monkeypatch.delenv("SIEGE_ALLOW_UNSAFE_PATHS", raising=False)
+        monkeypatch.setenv("SIEGE_TEST_VAR", "/etc")
+        default = Path.home() / "fallback"
+        result = _env_path("SIEGE_TEST_VAR", default)
+        assert result == default
+
+    def test_outside_home_accepted_with_opt_in(self, monkeypatch):
+        from siege_utilities.config.paths import _env_path
+        monkeypatch.setenv("SIEGE_ALLOW_UNSAFE_PATHS", "1")
+        monkeypatch.setenv("SIEGE_TEST_VAR", "/tmp/_pytest_unsafe_target")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/_pytest_unsafe_target"

(S108)